### ISSUE TITLE
bug: Pulumi config in docs contains invalid key

### ISSUE THREAD

**Issue by kralicky** [localstack/localstack#7059]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The pulumi config file provided in the integration docs [here](https://docs.localstack.cloud/integrations/pulumi/) is invalid, due to the `mobileanalytics` endpoint no longer existing.

When running `pulumi up` with this config file, it gives the following error:
```
    error: could not validate provider configuration: 1 error occurred:
        * Invalid or unknown key
```

Removing `mobileanalytics` from the endpoints config resolves this issue.

### Expected Behavior

The `mobileanalytics` endpoint should be omitted from the latest documentation, as it ostensibly no longer exists.

### How are you starting LocalStack?

Custom (please describe below)

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    Localstack Cockpit

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)
	
	`pulumi up`

### Environment

```markdown
- OS: Manjaro
- LocalStack: latest
```

**Comment by whummer** [comment]
Thanks for reporting @kralicky - our docs are open source, would you mind creating a small pull request against this repo? https://github.com/localstack/docs Thanks for your help! 🙌

**Comment by aadi58002** [comment]
I created a pull request for this issue in the docs repo `https://github.com/localstack/docs/pull/319`

**Comment by kralicky** [comment]
Sorry, wasn't able to get around to it. Thanks @aadi58002

**Comment by aadi58002** [comment]
> Sorry, wasn't able to get around to it. Thanks @aadi58002

You might want to close the issue as well

**Comment by whummer** [comment]
Thanks @aadi58002 @tuttinator , a PR with the fix has been merged and should be available with the latest version `0.5` of `pulumi-local`: 

```
pip install --upgrade 'pulumi-local>=0.5'
```

Please let us know if there are any other issues.. Thanks!
